### DISCUSSION TITLE
Searching/Showing Repository based on multiple lists, and excluding from few lists

### DISCUSSION THREAD

**Discussion by ajay-yug** [community/community#21035] [Other Feature Feedback, Questions, & Ideas]
**Searching/Showing Repository based on multiple lists, and excluding from few lists:**
This can be really helpful. 
One example can be: 
A developer wants to view projects from 'Android App Dev' list, and 'Time Management' list, excluding the ones which are 'Archived' list.

Also the list should be **sharable**, but by default be private. Similar to how we create Youtube playlists.

**Comment by rafaelteodoro8989** [comment]
jzjz
